lib/AssemblyUtil/FastaToAssembly.py

```diff
@@ -42,10 +74,16 @@ def import_fasta(self, params):
         mass_params = self._set_up_single_params(params)
         return self._import_fasta_mass(mass_params)[0]
 
-    def import_fasta_mass(self, params):
+    def import_fasta_mass(self, params, threads_per_cpu, max_threads, max_cumsize=None, parallelize=True):
         print('validating parameters')
         self._validate_mass_params(params)
-        return self._import_fasta_mass(params)
+        _validate_threads_param_input(threads_per_cpu, "THREADS_PER_CPU")
+        _validate_threads_param_input(max_threads, "MAX_THREADS")
+        params[_MCS] = _validate_max_cumsize(max_cumsize)
```

Comment:
I would pass this as a function argument rather than adding it to the user input parameters

Reply:
Duplicate with https://github.com/kbaseapps/AssemblyUtil/pull/89#discussion_r1357448645?

Reply:
No, that's talking about the threading params, this is talking about storing max_cumsize in `params`

Reply:
I see. Just line 82.

Reply:
Updated!